.github/workflows/ci.yml

```
@@ -128,11 +129,11 @@ jobs:
     needs: [lint, build-python]
     steps:
       - uses: actions/checkout@v3
-      # same as above
-      - name: Set up Python 3.10
+      # Nuitka currently only supports Python <=3.10 right now
```

```suggestion
      # Nuitka currently only supports Python <=3.10 right now, but their development branch has 3.11 support
```
why did i even think to mention this 😅